#!/usr/bin/env python
PACKAGE = "spencer_bagfile_tools"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("extra_calib_overall_multiplier", double_t, 1, "Additional overall multiplier for left and right tick count delta", 1, 0.5, 1.5)
gen.add("extra_calib_left_multiplier",    double_t, 1, "Additional multiplier for left tick count delta", 1, 0.5, 1.5)
gen.add("recalculate_path",               bool_t,   1, "Whether to recalculate entire path once a parameter has been changed.",  True)

gen.add("line_width",                     double_t, 0, "Line width multiplier for visualization", 1, 0.01, 10)
gen.add("arrow_length",                   double_t, 0, "Length of arrow head", 1, 0.01, 50)

gen.add("show_waypoints",                 bool_t,   0, "Show waypoints with timestamps every n seconds",  True)





exit(gen.generate(PACKAGE, "spencer_bagfile_tools", "ReconstructOdometry"))